core/src/main/java/org/apache/accumulo/core/client/mock/MockConfiguration.java

```diff
@@ -21,13 +21,16 @@
 
 import org.apache.accumulo.core.conf.AccumuloConfiguration;
 import org.apache.accumulo.core.conf.Property;
+import org.gaul.modernizer_maven_annotations.SuppressModernizer;
 
 import com.google.common.base.Predicate;
 
 /**
  * @deprecated since 1.8.0; use MiniAccumuloCluster or a standard mock framework instead.
  */
 @Deprecated
+@SuppressModernizer
+@SuppressWarnings("overloads")
```

Comment:
FYI this PropertyFilter will require a ```@SuppressModernizer``` since it prefers java.util.Predicate.  I assume you prefer this (just the one annotation here) as opposed to the two annotations over MockConfiguration that this will replace.

Reply:
My first preference is to just remove it (in violation of semver, with apologies in the release notes). My secondary preference would be to add the `@SuppressModernizer` like you mention.

Reply:
I think removing the method in MockConfiguration is OK.  It was deprecated and is now dangerous with the upgrade to Java 8, as shown by the warning.  Everything should still work the same in the Mock framework.  This will only break code that extended MockConfiguration.